config/page_length.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+  'row_limit' => '10',
+];
```

Comment:
cái nào dùng chung cho nhiều chỗ thì để trong file có tên là define.
em đổi tên file lại là `define` nha
<?php
return [
    //number of rows per page for pagination
    'page_length' => 10
];